setup_cfg_fmt.py

```diff
@@ -418,7 +418,8 @@ def format_file(
 
     # sort the classifiers if present
     if 'classifiers' in cfg['metadata']:
-        classifiers = sorted(set(cfg['metadata']['classifiers'].split('\n')))
+        classifiers = list(set(cfg['metadata']['classifiers'].split('\n')))
+        classifiers.sort(key=lambda s: [int(t) if t.isdigit() else t.lower() for t in re.split(r'(\d+)', s)])
```

Comment:
let's add a test case demonstrating this -- also probably worth splitting this out into its own function so it's more clear what's happening here

at the very least, the tests should demonstrate that the classifiers in the original issue sort properly